applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -196,10 +196,11 @@ config BUF_BLE_RX_PACKET_NUM
 	  gaps due to BLE retransmits.
 
 config STREAM_BIDIRECTIONAL
-	bool "Enable bi-directional stream - Currently not supported"
+	depends on TRANSPORT_CIS
+	bool "Enable bidirectional stream"
```

Comment:
Please remove `enable` as per guide lines:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Reply:
Done